travel-journal/index.md

```diff
@@ -0,0 +1,910 @@
+---
+title: 'Personal Travel Journal with Vue.js and Directus'
+description: 'This tutorial covers how to create a personal travel journal with Directus and Vue.js with authentication features'
+author:
+  name: 'Mercy bassey'
+  avatar_file_name: 'mercy-bassey.jpg'
+---
+
+## Introduction
+
+Vue.js, a progressive JavaScript framework known for its simplicity and flexibility, has gained widespread popularity among developers for building interactive user interfaces. In this tutorial, you we’ll build a personal travel journal application using Vue.js in conjunction with Directus, an open-source headless CMS. In this guide we’ll go through the process of implementing user authentication with Directus and demonstrate how to interact with a Directus collection using the Vue.js framework.
+
+## Before You Start
+
+You'll need:
+
+- [Node.js](https://nodejs.org/en/) and a [text editor](https://kinsta.com/blog/best-text-editors/) installed on your machine.
```

Comment:
```suggestion
- [Node.js](https://nodejs.org/en/) and a code editor installed on your machine.
```

No need to link to Kinsta. Also specifying 'code' editor because that's ideally what they'll use.